1.data_preprocessing/1.make_pseudobulks.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<p>this all looks great! My only suggestion is to make some plots of the raw data before it is pseudobulked, so that you get a deeper understanding of the data and can see if there are any issues with the data that may affect you interpretation of it later after processing / after making pseudobulks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>